src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static int threshold;
```

Comment:
not worth keeping as a field, it's value is read only once - in resize. You can calculate it based off other known variables

Reply:
threshold *= 2; 
how can this field be changed if I remove that field?